src/ui/public/agg_types/metrics/std_deviation.js

```diff
@@ -11,19 +11,18 @@ export default function AggTypeMetricStandardDeviationProvider(Private) {
       return details.valProp;
     },
     makeLabel: function () {
-      let details = this.keyedDetails(this.params.customLabel)[this.key];
-      return details.title + ' of ' + this.fieldDisplayName();
+      return this.keyedDetails(this.params.customLabel, this.fieldDisplayName())[this.key].title;
```

Comment:
I would feel more comfortable if this used `get(keyedDetails, [this.key, 'title'], '--')` or something similar, but since the `keyedDetails()` function is defined so close by it **probably** won't be a big deal.